packages/application-config/src/process-config.ts

```diff
@@ -104,8 +117,14 @@ const processConfig = ({
   // This is important to determine to which URL the MC should redirect to
   // after successful login.
   const applicationId = isProd
-    ? `${customApplicationData.id}:${customApplicationData.entryPointUriPath}`
-    : `__local:${customApplicationData.entryPointUriPath}`;
+    ? `${configurationData.id}:${
+        (configurationData as CustomApplicationData).entryPointUriPath ||
+        'custom-view'
```

Comment:
Custom Views does not have an entry point, so we use a placeholder to keep using the same application identifier value pattern.

Reply:
Yes, I thought about it but didn't want to deviate too much and adjusting things here and there seemed fine to me, but I agree it would be cleaner.
Let me try to refactor this part.